app/build.gradle

```diff
@@ -4,15 +4,19 @@ apply plugin: 'kotlin-android'
 
 apply plugin: 'kotlin-android-extensions'
 
+androidExtensions {
+    experimental = true
+}
+
 android {
-    compileSdkVersion 27
+    compileSdkVersion 28
     defaultConfig {
         applicationId "com.arctouch.codechallenge"
         minSdkVersion 19
-        targetSdkVersion 27
+        targetSdkVersion 28
         versionCode 1
         versionName "1.0"
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
Why did you choose to use AndroidX?

Reply:
Because I have updated all the libs in the project and all new Support Library development will occur in the AndroidX library.

https://developer.android.com/jetpack/androidx/